load-testing/tests/relays_stress_test.go

```diff
@@ -381,6 +438,7 @@ func (s *relaysSuite) ALoadOfConcurrentRelayRequestsAreSentPerSecondAsFollows(ta
 
 	// tickerCircuitBreaker is used to limit the concurrency of batches and error
 	// if the limit would be exceeded.
+	// TODO_DISCUSS: Are we really going to have concurrent batches?
```

Comment:
My thinking was that a batch shouldn't be blocked by a previous batch, which could be possible in the event of a long-tail. If a batch can be blocked then there's no guarantee that the correct load parameters are being used at the expected time. 

This was an attempt to mitigate that concern and gives us control over the maximum time to wait for a batch to complete (in terms of # of concurrent batches).